### DISCUSSION TITLE
Cannot connect to snowflake - driver issue

### DISCUSSION THREAD

**Discussion by techlock77** [dbeaver/dbeaver#14186] [Q&A]
#### System information: 
- Operating system (distribution) and version - WIN 10
- DBeaver version - 21.1.2.202107041908
- Additional extensions

#### Connection specification:
- Database name and version - Snowflake
- Driver name - 
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
Cannot connect to the snowflake db, due to some driver libraries. See snapshot below.
![image](https://user-images.githubusercontent.com/66756251/133735197-79ba2915-28bd-4875-b9b4-86db239c6d79.png)

####Below is the error description:

Can't create driver instance
  Error creating driver 'Snowflake' instance.
Most likely required jar files are missing.
You should configure jars in driver settings.

Reason: can't load driver class 'net.snowflake.client.jdbc.SnowflakeDriver'
  Error creating driver 'Snowflake' instance.
Most likely required jar files are missing.
You should configure jars in driver settings.

Reason: can't load driver class 'net.snowflake.client.jdbc.SnowflakeDriver'
    net.snowflake.client.jdbc.SnowflakeDriver
    net.snowflake.client.jdbc.SnowflakeDriver

**Comment by techlock77** [comment]
@uslss Can you please steer me in the right direction.

Thanks for your help in advance.

**Comment by LonwoLonwo** [comment]
Hello @techlock77 
It can be a problem with the internet connection.
These links can be helpful:

https://github.com/dbeaver/dbeaver/issues/11862#issuecomment-806441885
https://github.com/dbeaver/dbeaver/issues/9812#issuecomment-878181728
